data/mods/realguns/ammo.json

```diff
@@ -355,6 +355,7 @@
     "casing": "410_hull",
     "range": 5,
     "damage": 32,
+    "dispersion": 800,
```

Comment:
Wait a second, how come shotgun rounds now have absurdly high dispersion, when most of the ammo here has under 100?

Reply:
@smolbird https://github.com/CleverRaven/Cataclysm-DDA/pull/23720#issuecomment-389651487

Reply:
I'm reconsidering the dispersion and range of shotguns right now.

Reply:
If you're tweaking shotgun accuracy more, please add some tests for it in tests/ranged_accuracy.cpp and make them pass.

Reply:
@Firestorm01X2 800 dispersion is obscene, where on earth did you get that number. Because that is "literally can't hit the broad side of a barn" levels for shot which is supposed to be EASIER to score a hit with.

Reply:
@smolbird  Is I said here: shotguns now has range of [0 , 6] and designed for very close range shooting. So the must scale between [0, 6] tiles. But in my PR I've reduced shotgun dispersion and sight dispersion. It lead to: shotgun alway hit in the head.

To do something with that I've artificially add +800 to shotgun ammo to get them back to [0, 6]. That 800 was determing by observing disperion ranges and simpe "let' try to do it that way".